### PR TITLE
[Mailer] Configuration of the message_bus to use

### LINKED ISSUE #12736: [Mailer] Allow to configure or disable the message bus to use
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#34648
| PR author(s) | @ogizanagi
| Merged in    | 5.1-dev

### PR DESCRIPTION
**Pull request by t-richard**
Documents symfony/symfony#34648

Closes #12736

### CODE REVIEW

**Review by Nyholm — APPROVED** [review]
I though this was document already. 

Well done!

**Comment by t-richard** [comment]
I'm currently going through the `5.1` milestone in the issues and trying to document missing features listed there, that's how I caught it :slightly_smiling_face:

**Comment by Nyholm** [comment]
Nice. You are the best!

That is very appreciated and a great way learn about (new) features.

**Comment by t-richard** [comment]
Actually got the idea from you [some months ago](https://twitter.com/TobiasNyholm/status/1314478224286650368) :wink: 

And had some time lately to contribute !

**Review by maxhelias — APPROVED** [review]

File: mailer.rst [hunk 1]
```diff
@@ -1094,6 +1094,48 @@ you have a transport called ``async``, you can route the message there:
...
+
+    .. code-block:: php
+
+        // config/packages/mailer.php
+        $container->loadFromExtension('framework', [
+            'mailer' => [
+                'message_bus' => 'app.another_bus',
+            ],
+        ]);
+
```
Comment (OskarStark) [inline mailer.rst:1138]:
```suggestion

.. versionadded:: 5.1

    The `message_bus` option was introduced in Symfony 5.1.

```

**Review by OskarStark — CHANGES_REQUESTED** [review]
Please add missing version added directive, thanks

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you Thibault.

**Pull request merged**